secrets/secrets.nix

```diff
@@ -34,6 +34,8 @@ in
   # `CLOUDFLARE_DNS_API_TOKEN=...`
   "cloudflare-dns-api-token.env.age".publicKeys = sysadmins ++ [ ares ];
 
+  "restic-s3-repo-pwd-ares.age".publicKeys = sysadmins ++ [ ares ];
```

Comment:
I guess you're right in that if it's ever stolen, it would be from the host itself, which would mean attackers would also be able to exfiltrate a second password just as easily.

It still feels wrong intuitively to reuse passwords, but I'll change it in the name of simplicity and so we don't have a thousand agenix secrets.